src/quest_manager/models.py

```diff
@@ -975,3 +988,55 @@ def get_minutes_to_complete(self):
 
         minutes = (self.first_time_completed - self.timestamp).total_seconds() / 60
         return minutes
+
+
+class QuestionType(models.Model):
+    id = models.CharField(
+        max_length=100,
+        primary_key=True,
+        help_text="Short, memorable, unique identifier for a question type e.g. 'short_answer', 'file_upload', 'multiple_choice'.",
+        serialize=False
+    )
+    name = models.CharField(
+        max_length=255,
+        help_text="The name of the question type e.g. 'Short Answer', 'File Upload', 'Multiple Choice'."
+    )
+    description = models.TextField(blank=True, null=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Question(models.Model):
+    instructions = models.TextField()
+    solution_text = models.TextField(null=True, blank=True)
+    solution_file = RestrictedFileField(blank=True, null=True, upload_to='quest_questions/')
+    marker_notes = models.TextField(null=True, blank=True)
+
+    required = models.BooleanField(default=True)
+    datetime_created = models.DateTimeField(auto_now_add=True)
+    datetime_last_edit = models.DateTimeField(auto_now=True)
+    ordinal = models.PositiveIntegerField(default=1)
+
+    quest = models.ForeignKey(Quest, on_delete=models.CASCADE)
+    type = models.ForeignKey(QuestionType, on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
Why null=true blank=true?  What would it mean for a question to not have a type?

Reply:
My mistake. I was getting Django migration errors, and it was telling me that I had to either provide a default value or allow null values. I changed it so that now the default value is 'short_answer'.

Reply:
Is this because your dev DB has objects with no type, from earlier migrations?  We shouldn't need to provide a type (just as you don't provide a quest).

Might just be a hazard of re-migrating the same thing multiple timeswith different parameters.  Your DB doesn't get reset, so might be good to refresh your db, i.e start a new one on the dev branch, then switch over to this branch and run migrations, so it more closely matches the state of when these migrations would actually happen.